pepc.html

```diff
@@ -8,14 +8,48 @@
     <link rel="shortcut icon" href="favicon.ico" />
     <link rel="apple-touch-icon" href="app-icon.png" />
     <link rel="stylesheet" href="style.css" />
+    <script src="pepc.js"></script>
   </head>
   <body>  
     <div class="before"></div>
     <div class="content">      
-      <div><permission id="camera" type="camera"></permission></div>
-      <div><permission id="microphone" type="microphone"></permission></div>
-      <div><permission id="camera-microphone" type="camera microphone"></permission></div>
-      <div><permission id="geolocation" type="geolocation"></permission></div>
+      <span class="permission-element-box">
+        <div><permission id="geolocation" type="geolocation"></permission></div>
+        <div class="access-status">
+          Access status: <span id="geolocation-access-status">⚫️ unknown</span>
+        </div>
+        <div class="access-status">
+          Error message: <span id="geolocation-error-message">N/A</span>
+        </div>
+      </span>
+      <span class="permission-element-box">
+        <div><permission id="camera" type="camera"></permission></div>
+        <div class="access-status">
+          Access status: <span id="camera-access-status">⚫️ unknown</span>
+        </div>
+        <div class="access-status">
+          Error message: <span id="camera-error-message">N/A</span>
+        </div>
+      </span>
+      <span class="permission-element-box">
+        <div><permission id="microphone" type="microphone"></permission></div>
+        <div class="access-status">
+          Access status: <span id="microphone-access-status">⚫️ unknown</span>
+        </div>
+        <div class="access-status">
+          Error message: <span id="microphone-error-message">N/A</span>
+        </div>
+      </span>
+      <span class="permission-element-box">
+        <div><permission id="camera-microphone" type="camera microphone"></permission></div>
+        <div class="access-status">
+          Access status:
+          <span id="camera-microphone-access-status">⚫️ unknown</span>
+        </div>
+        <div class="access-status">
+          Error message: <span id="camera-microphone-error-message">N/A</span>
+        </div>
+      </span>  
       <div class="demo-instructions">
         <h2>👩🏻‍💻 Demo instructions</h2>
         <p>You will see empty rectangles unless this browser implementes basic support for the &lt;permission&gt;&nbsp;element.</p>

```

Comment:
Good chance to fix type typo (or remove this if we add placeholders, see above).
```suggestion
        <p>You will see empty rectangles unless this browser implements basic support for the &lt;permission&gt;&nbsp;element.</p>

```

Reply:
I went with the placeholders/fallbacks.